src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -88,6 +88,38 @@ private StringEscapeCharacterFormatDefinition()
             }
         }
 
+        #endregion
+        #region Keyword - Control
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.ControlKeyword)]
+        [Name(ClassificationTypeNames.ControlKeyword)]
+        [Order(After = ClassificationTypeNames.Keyword)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class ControlKeywordFormatDefinition : ClassificationFormatDefinition
+        {
+            private ControlKeywordFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.Keyword_Control;
+            }
+        }
+        #endregion
+        #region Operator - Overload
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.OperatorOverload)]
+        [BaseDefinition(PredefinedClassificationTypeNames.Operator)]
+        [Name(ClassificationTypeNames.OperatorOverload)]
+        [Order(After = PredefinedClassificationTypeNames.Operator)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class OperatorOverloadFormatDefinition : ClassificationFormatDefinition
+        {
+            private OperatorOverloadFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.Operator_Overload;
```

Comment:
maybe `Operator (overloaded)`?

Reply:
Maybe, I was basing it on how we named `String - Verbatim`. I'll add this to my list of design questions.